firmware/controllers/gauges/tachometer.cpp

```diff
@@ -18,26 +18,71 @@
 EXTERN_ENGINE;
 
 static scheduling_s tachTurnSignalOff;
+static float halfperiodMs = 4; /* period in ms, used for multi pulse output, actually half period */
+static uint8_t isStarted = 0;
+
+static void toggleTachPinHigh(void);
+static void toggleTachPinLow(void);
 
 static void turnTachPinLow(void) {
 	enginePins.tachOut.setLow();
 }
 
+static void toggleTachPinLow(void)
+{
```

Comment:
code style - note how the rest of the firmware define methods

Reply:
Do you mean by coding style? use it as suggested here, https://rusefi.com/wiki/index.php?title=Development:Code_Style